pkgs/top-level/python-packages.nix

```diff
@@ -2964,6 +2964,33 @@ in {
     };
   };
 
+  httpserver = buildPythonPackage rec {
+    name = "${pname}-${version}";
+    pname = "httpserver";
+    version = "1.1.0";
+    buildInputs = with self; [ docopt ];
+    doCheck = false;
```

Comment:
why do you disable the tests? If you disable the tests, always include a comment.